packages/docz/src/components/ErrorReporter.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { SFC } from 'react'
+
+interface ErrorInfo {
+  componentStack: string
+}
+
+export interface ErrorReporterProps {
+  error: Error
+  errorInfo: ErrorInfo
+}
+
+const ErrorReporter: SFC<ErrorReporterProps> = ({ error, errorInfo }) => {
+  return (
+    <div>
+      <p>An error occurred while rendering this component:</p>
+      <p>{error.message}</p>
+      <pre>{errorInfo.componentStack}</pre>
+    </div>
+  )
+}
+
+export default ErrorReporter
```

Comment:
For some reasons, when you use typescript is a better pattern don't use default exports. So, try to export directly your components instead of make default exports

```js
import { ErrorReporter } from './ErrorReporter'
```